### ISSUE TITLE
Few openebs components still using ubuntu:16.04 as base image 

### ISSUE THREAD

**Issue by jj-2020** [openebs/openebs#3386]
## Description
Few components like cstor-volume-manager and cstor-csi-plugin seem to be using ubuntu:16.04 as base image.  According to below document, ubuntu 16.04 has reached it's EOL-
https://wiki.ubuntu.com/Releases

## Possible Solution
Update the base images

## Steps to Reproduce
https://github.com/openebs/cstor-operators/blob/master/build/volume-manager/Dockerfile#L17
https://github.com/openebs/cstor-csi/blob/master/buildscripts/cstor-csi-driver/Dockerfile#L16

**Comment by abhiTamrakar** [comment]
+1 since we are evaluating openebs as well.
can we use debian images instead of ubuntu, they have lesser EOL kindof problems?

**Comment by kmova** [comment]
We are in the process of upgrading the images to latest ubuntu as part of 2.9. 

Will follow-up on this issue with the status after the next community sync-up with the contributors.

**Comment by abhiTamrakar** [comment]
@kmova these are EOL images, would produce N number of un-patchable vulnerabilities, would it be possible to have the fix made available ASAP? may be backported to 2.8 as well. Just a request.

**Comment by abhiTamrakar** [comment]
@kmova any updates, when do we get the new images? We cannot use vulnerable images.

**Comment by kmova** [comment]
The following images are available now @abhiTamrakar. The 2.9.0 images will be generated on 15th. 

```
✔ openebs/linux-utils:2.9.0-RC2
✔ openebs/cstor-pool:2.9.0-RC2
✔ openebs/cstor-istgt:2.9.0-RC2
✔ openebs/jiva:2.9.0-RC2
✔ openebs/m-apiserver:2.9.0-RC2
✔ openebs/cstor-pool-mgmt:2.9.0-RC2
✔ openebs/cstor-volume-mgmt:2.9.0-RC2
✔ openebs/m-exporter:2.9.0-RC2
✔ openebs/m-upgrade:2.9.0-RC2
✔ openebs/admission-server:2.9.0-RC2
✔ openebs/openebs-k8s-provisioner:2.9.0-RC2
✔ openebs/snapshot-controller:2.9.0-RC2
✔ openebs/snapshot-provisioner:2.9.0-RC2
✔ openebs/velero-plugin:2.9.0-RC2
✔ openebs/provisioner-localpv:2.9.0-RC2
✔ openebs/cstor-csi-driver:2.9.0-RC2
✔ openebs/cvc-operator:2.9.0-RC2
✔ openebs/cspc-operator:2.9.0-RC2
✔ openebs/cstor-volume-manager:2.9.0-RC2
✔ openebs/cstor-pool-manager:2.9.0-RC2
✔ openebs/cstor-webhook:2.9.0-RC2
✔ openebs/upgrade:2.9.0-RC2
✔ openebs/migrate:2.9.0-RC2
✔ openebs/jiva-operator:2.9.0-RC2
✔ openebs/zfs-driver:1.7.0-RC2
✔ openebs/node-disk-manager:1.4.1-RC2
✔ openebs/node-disk-operator:1.4.1-RC2
✔ openebs/node-disk-exporter:1.4.1-RC2
✔ openebs/lvm-driver:0.5.0-RC2
```

**Comment by prateekpandey14** [comment]
@abhiTamrakar thanks for pointing out, issue has been fixed as part of v2.9.0 openebs release.
